contracts/ccip/ccip_offramp/sources/offramp.move

```diff
@@ -83,9 +83,9 @@ public struct Any2SuiRampMessage has drop {
     header: RampMessageHeader,
     sender: vector<u8>,
     data: vector<u8>,
-    receiver: address, // this is the message receiver
+    receiver: address, // package ID for contract calls
     gas_limit: u256,
-    token_receiver: address,
+    token_receiver: address, // user address for token transfers
```

Comment:
i don't think this comment is very clear. token receiver can be an EOA or an object

Reply:
Let me remove this